src/components/views/elements/MemberEventListSummary.js

```diff
@@ -145,9 +145,13 @@ module.exports = React.createClass({
     },
 
     shouldComponentUpdate: function(nextProps, nextState) {
+        // Update if
+        //  - The number of summarised events has changed
+        //  - or if the summary is currently expanded
+        //  - or if the summary is about to toggle to become collapsed
         return (
             nextProps.events.length !== this.props.events.length ||
-            nextState.expanded !== this.state.expanded
+            this.state.expanded || nextState.expanded
```

Comment:
I thought we auto-expanded if there were "few" events?

So we should treat "few" events the same as "state.expanded"